BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/ble/beacon/signal/ArmaFilter.java

```diff
@@ -30,47 +29,49 @@
      **/
     private static float DEFAULT_ARMA_FACTOR = 0.95f;
 
-    private float armaFactor;
+    private float armaFactor = DEFAULT_ARMA_FACTOR;
     private float armaRssi;
     private boolean isInitialized = false;
 
-    public ArmaFilter(long duration, TimeUnit timeUnit) {
-        super(duration,timeUnit);
-        this.armaFactor = DEFAULT_ARMA_FACTOR;
+    public ArmaFilter() {
```

Comment:
You will not be able to create an `ArmaFilter` with custom `duration` and `timeUnit` on construction.